pubsub/cloud-client/quickstart/pub.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START pubsub_quickstart_pub_all]
+import argparse
+import time
+# [START pubsub_quickstart_pub_deps]
+from google.cloud import pubsub_v1
+# [END pubsub_quickstart_pub_deps]
+
+
+def get_callback(api_future, data):
+    """Wraps message data in the context of the callback function."""
+    def callback(api_future):
+        if api_future.exception():
+            print("There was a problem with message {}".format(data))
```

Comment:
This should re-raise the exception. Otherwise we get into the "Marshall problem."

Reply:
Done.